src/components/pages/List.tsx

```diff
@@ -3,11 +3,11 @@ import Header from '../organisms/Header';
 import {GalleryList} from '../organisms/Gallerylist';
 
 interface urlProps{
-  url: number,
+  listnum: number,
```

Comment:
list numberで２語なので以下ですかね
```suggestion
  listNum: number,
```

Reply:
ご指摘いただきありがとうございます。
修正したものをpushいたしました。